content/docs/rfcs/67/README.md

```diff
@@ -0,0 +1,206 @@
+---
+slug: 67
+title: 67/STATUS-WAKU2-USAGE
+name: Status Waku2 Usage
+status: raw
+category: Standards Track
+description: Get familiar with all the Waku protocols used by the Status application.
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors: 
+- Jimmy Debe <jimmy@status.im>
+
+---
+
+# Abstract
+
+Status is a chat application which has several features, including, but not limited to -
+- Private 1:1 chats, described by [55/STATUS-1TO1-CHAT](/spec/55)
+- Large scale group chats, described by [56/STATUS-COMMUNITIES](/spec/56)
+
+This specification describes how a Status implementation will make use of the underlying infrastructure, 
+Waku, which is described in [10/WAKU2](/spec/10).
+
+# Background 
+
+The Status application aspires to achieve censorship resistance and incorporates specific privacy features, 
+leveraging the comprehensive set of protocols offered by Waku to enhance these attributes. 
+Waku protocols provide secure communication capbailites over decentralized networks. 
+Once integrated, an application will benifit from privacy preserving, 
+censorship resistance and spam protected communcation. 
+
+Since Status uses a large set of Waku protocols, 
+it is imperative to describe how each are used. 
+
+# Terminology
+
+| Name  | Description |
+| --------------- | --------- |
+| `RELAY`| This refers to the Waku Relay protocol, described in [11/WAKU2-RELAY](/spec/11) |
+|`FILTER` | This refers to the Waku Filter protocol, described in [12/WAKU2-FILTER](/spec/12) |
+| `STORE` | This refers to the Waku Store protocol, described in [13/WAKU2-STORE](/spec/13) |
+| `MESSAGE` | This refers to the Waku Message format, described in [14/WAKU2-MESSAGE](/spec/14) |
+| `LIGHTPUSH` | This refers to the Waku Lightpush protocol, described in [19/WAKU2-LIGHTPUSH](/spec/19) |
+| `Pubsub Topic` / `Content Topic` | This refers to the routing of messages within the Waku network, described in [23/WAKU2-TOPICS](/spec/23/) |
+
+
+### Waku Node:
+
+A server running Waku software configured with a set of protocols.
+A Status client is a Waku node as that can be a full relay node or
```

Comment:
Some specs like [12/WAKU2-FILTER](https://rfc.vac.dev/spec/12/#design-requirements), in [10/WAKU2](https://rfc.vac.dev/spec/10/#security-considerations) it is mentioned once in the Security Considerations. In [6/WAKU1](https://rfc.vac.dev/spec/6/) "A full bloom filter (all the bits set to 1) means that the node is to be considered a `Full Node` and it will accept any topic."

The language of a Full node is not consistent throughout the specifications.

Reply:
cc: @jm-clius

Reply:
We shouldn't refer to any `WAKU1` specs or terminology as this has been replaced by new concepts for `WAKU2`.

In Waku we generally don't refer to "full nodes" or "light nodes" with specific definitions. Instead, we have adaptive nodes that can be as "full" or "light" as is required by their application/environment. When a node is "resource restricted" it tends to choose protocols suitable for the constraints it faces, but there is no definition of what set of protocols constitutes a "light" mode. My reading of "full" or "light" in this RFC, though, was that these were Status concepts with clearly defined use case of Waku in each case? In other words, I'll opt for the terminology that Status uses inside their app when switching from "default relay" mode to filter+lightpush mode.